adam-core/src/test/scala/org/bdgenomics/adam/converters/AlignmentRecordConverterSuite.scala

```diff
@@ -249,7 +249,7 @@ class AlignmentRecordConverterSuite extends FunSuite {
       .split('\n')
 
     assert(!secondRecord.getReadMapped)
-    assert(!secondRecord.getReadNegativeStrand)
```

Comment:
Given the input data, this seems likely a bug in the test that was previously covered up by fact that for unmapped reads the code previously did not set this flag even if present in input.